docs/docs/README.md

```diff
@@ -9,12 +9,12 @@ slug: /
 
 Main features:
 
-- All-platform user sign-in / sign-up experience with SDK support (SPA, iOS and Android)
-- Passwordless (SMS / Email) support
-- Out-of-box social sign in support (GitHub, Google, WeChat, Alipay, etc.)
+- An OIDC-based identity service
+- All-platform user sign-in/up experience with dark mode and SDKs (Web, iOS, and Android)
+- Sign-in/up with dynamic SMS/Email passcode
+- Out-of-box social sign-in integration (GitHub, Google, WeChat, Alipay, etc.)
```

Comment:
nit: `Alipay, etc.` → `Alipay, and etc.`

Reply:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/14722250/173995842-67905b8f-7f67-4728-abb7-8094abfaa036.png">